fe1-web/src/core/objects/Timestamp.ts

```diff
@@ -88,4 +90,37 @@ export class Timestamp extends Number implements Number {
   public addSeconds(seconds: number): Timestamp {
     return new Timestamp(this.valueOf() + seconds);
   }
+
+  /**
+   * Returns *some* string representation of this object.
+   * If you need access to the unterlying data type use .valueOf() and
+   * if you want to serialize an instance use .toState() instead
+   */
+  public toString(): string {
+    return super.toString();
+  }
```

Comment:
if we want a `toString` representation, I'd be almost tempted to make it an ISO-8601 representation 😅

Reply:
👍👍👍😁

Reply:
Okay just realized that this results in a problem with a lot of hash computations which would have to be changed to something else 😂 Hence we still need access to `valueOf()` in one way or another 🤔

Reply:
fromStringArray() could for instance also allow Timestamps and convert them using `valueOf()` ..? Probably the cleanest solution?